Frameworks/UIKit/UIButton.mm

```diff
@@ -719,6 +747,24 @@ - (void)touchesMoved:(NSSet*)touchSet withEvent:(UIEvent*)event {
     [super touchesMoved:touchSet withEvent:event];
 }
 
+// Updates our highlighted state as needed
+- (void)_updateHighlightedState {
+    [self setHighlighted:(_isInTouchSequence || _isXamlElementPressed)];
```

Comment:
>_isXamlElementPressed [](start = 48, length = 21)

This method needs to be triggered by IsPressed changed; but can't this just access the xaml button's IsPressed property directly?